src/lib.rs

```diff
@@ -145,12 +148,16 @@ fn check_egui_wants_focus(
 
 fn do_camera_zoom(
     mut query: Query<(&PanCam, &Camera, &mut Projection, &mut Transform)>,
+    mut pinch_events: EventReader<PinchGesture>,
     scroll_events: EventReader<MouseWheel>,
     primary_window: Query<&Window, With<PrimaryWindow>>,
 ) {
+    const PINCH_MULTIPLIER: f32 = 1000.;
```

Comment:
I think the naming of these two constants could be improved slightly. Also, it's a bit strange to scale it down by 1000 and then up reverse it later. 

I think scroll sensitivity and pinch sensitivity would be good names, and handle both senaitivity factors here. And rename scroll_offset to zoom_delta, perhaps?

What do you think?

Reply:
I agree it could be reworked to be a little clearer, will adjust later today.

Reply:
I've taken some liberty to make a larger refactor, and also added configuration options for pinch and mouse wheel zoom sensitivity.

Let me know what you think, happy to adjust.